src/main/java/org/zalando/nakadi/domain/SubscriptionListWrapper.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.nakadi.domain;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Collections;
+import java.util.List;
+
+@Immutable
+public class SubscriptionListWrapper {
```

Comment:
And what do you think about adding pagination. I know this is not in the original scope for this PR so I would be ok with adding it next.